pycds/orm/views/version_84b7fc2596d5.py

```diff
@@ -193,6 +193,12 @@ class ObsWithFlags(Base, ReplaceableView):
     This view joins Obs with History and Variable.
     """
 
+    """ 
+    The column net_var_name is defined as a String, but the source 
+    is actually of type CIText 
+    """
+
+
```

Comment:
Good comment, but I don't think _two_ docstrings is idiomatic Python. Make it a regular comment, and consider putting it next to the column definition. Also, if you're not familiar with them, have a quick look at what docstrings are.